### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by dannert**
This pull request includes the newly added files to implement Oracle Database as a Service on PowerVC/AIX, including the required IBM Cloud Manager Templates to deploy via CAM or from ICP catalog.
All content is grouped under a new folder named: oracle.

### CODE REVIEW

File: terraform/oracle/cam-ora/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# cam-demo
+Demonstrate IBM Cloud Automation Manager functionality to deploy orchestrated VMs to IBM PowerVC
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/README.md]: Nit - period at the end

File: terraform/oracle/cam-ora/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# cam-demo
+Demonstrate IBM Cloud Automation Manager functionality to deploy orchestrated VMs to IBM PowerVC
+
+This project builds on the Proof of Technology described in “Deploying Oracle® Database as a Service with IBM Cloud PowerVC Manager" where we demonstrate how IBM Cloud PowerVC Manager (PowerVC) technology enables the implementation of Database-as-A-Service (DBaaS) for an Oracle® database. See sister project: *powervc-oradbaas*.
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/README.md]: Can you include a link in the markdown?

File: terraform/oracle/cam-ora/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+We then expand the scope and illustrate the workload orchestration capabilities of CAM by creating a deployable service which co-deploys two virtual machines (VM) - a database server and an application server. The result of that orchestrated deployment is a running and pre-loaded oracle database and a workload driver in the second VM ready for the end-user to connect to and drive simulated transactions against the database. The template related files for this part can be found in: *oraclient* and, for the service definition, in *oradbsc*.
+
+Familiarity with IBM Cloud Private v3.2, IBM Cloud Automation Manager v3.2, IBM Power environment, IBM Power Virtualization Center (PowerVC), AIX and Linux system dministration is assumed. The creation of the deployable image for the Oracle database server, as described in *powervc-oradbaas*, requires working knowledge of Oracle 12c database software.
+
+# Prerequisites
+* Installed and working IBM PowerVC 1.3.1 or later install with IBM Power servers and PowerVC vontrolled storage
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/README.md]: s/vontrolled/controlled

File: terraform/oracle/cam-ora/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* Deployable Oracle DBaaS image in PowerVC and created as described in sister project powervc-oradbaas
+* Installed ICP 3.2 on IBM Power
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/README.md:15]: We should be basing things off OCP at this time

File: terraform/oracle/cam-ora/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* Installed IBM Cloud Automation Manager 3.2.x
+* Linux on Power deployable image
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/README.md]: Any particular distro needed?

File: terraform/oracle/cam-ora/oraclient/main.tf [hunk 6]
```diff
@@ -0,0 +1,151 @@
+######################################################################
+#  Copyright: IBM Corp., 2019
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/oraclient/main.tf:2]: Apache2 license?

File: terraform/oracle/cam-ora/oradbaas/main.tf [hunk 7]
```diff
@@ -0,0 +1,27 @@
+#####################################################################
```
Comment (jwcroppe) [inline terraform/oracle/cam-ora/oradbaas/main.tf:21]: Should have consistent headers across TF files... and just generically licensed under Apache 2?

File: terraform/oracle/cam-ora/README.md (hunk 4 above)
Comment (dannert) [reply terraform/oracle/cam-ora/README.md:15]: I agree, but OCP on Power has been very slow moving --> this whole Oracle DBaaS and CAM project was implemented and tested with ICP. It significantly precedes any Openshift on Power MCM/CAM availability.
I'm working on building a Power OCP 3.1.1 cluster into which I plan to deploy CAM, retest the templates and service and likely provide an updated document.

File: terraform/oracle/cam-ora/oraclient/main.tf (hunk 6 above)
Comment (dannert) [reply terraform/oracle/cam-ora/oraclient/main.tf:2]: Do you have a "boiler plate" I should use for the header?

File: terraform/oracle/cam-ora/oradbaas/main.tf (hunk 7 above)
Comment (dannert) [reply terraform/oracle/cam-ora/oradbaas/main.tf:21]: You mean all tf files should have the exact same header? I can do that but wait for your "boiler plate" with Apache 2 license text - or is the LICENSE file at the top of your project sufficient and I should just remove the "IBM Copyright" ?

**Review by jwcroppe — APPROVED** [review]

File: terraform/oracle/cam-ora/oraclient/main.tf (hunk 6 above)
Comment (dannert) [reply terraform/oracle/cam-ora/oraclient/main.tf:2]: Updated all tf and code files to show Apache 2.0 license

File: terraform/oracle/cam-ora/oradbaas/main.tf (hunk 7 above)
Comment (dannert) [reply terraform/oracle/cam-ora/oradbaas/main.tf:21]: Updated all tf and code files to show Apache 2.0 license

**Pull request merged**